src/flavors/__tests__/__snapshots__/openapi.test.ts.snap

```diff
@@ -145,21 +162,14 @@ Object {
         },
         "required": Array [
           "requiredBooleanValue",
+          "requiredDateStringValue",
           "requiredDateValue",
           "requiredEnumValue",
           "requiredIntegerValue",
           "requiredNestedValue",
           "requiredNumberValue",
           "requiredStringValue",
           "requiredUUIDValue",
-          "optionalBooleanValue",
```

Comment:
I found that OpneAPI decorators were not using the optional flag, so included a fix in this PR.